server/app/routes/v1/services_api.rb

```diff
@@ -16,10 +16,16 @@ class ServicesApi < Roda
       validate_access_token
```

Comment:
(these by the way are no longer needed unless the current path is in the auth middleware's "soft exclude" list, otherwise the middleware will halt the request before it reaches here)